### PR TITLE
Topics/k1ch/ Admin DELETE:/personas/{persona_key}

### PR DESCRIPTION
**Pull request by k1ch**
# Overview
- [feat: topics/k1ch/admin-delete-personas / Implement API](https://github.com/DMGT-TECH/the-usher-server/commit/0e579ba892052185bd5f5d5afe3f9e1a436e9797)
- [chore: topics/k1ch/admin-delete-personas / API and DB tests](https://github.com/DMGT-TECH/the-usher-server/pull/90/commits/4a8b7660b4645f5d5daf5a373ba2e49cb2a3b18f)

<details open>
  <summary> Added OAS: </summary> 

image

</details>

<details open>
  <summary> Added Tests: </summary> 

image

</details>

## Your PR Checklist 🚨

❤️ Please review the [guidelines for contributing](../docs/CONTRIBUTING.md) to this repository. Our goal is to merge PRs fast 💨 .

- [x] Check your code additions will fail neither code linting checks nor unit tests.
- [x] Additional units tests have been added to prove code updates work and fixes are effective
- [x] Additional documentation has been added (if appropriate)
- [x] Update Assignee field to yourself

## Issue Reference

Relates to https://github.com/DMGT-TECH/the-usher-server/issues/59

### CODE REVIEW

File: server/the-usher-openapi-spec.yaml [hunk 1]
```diff
@@ -469,6 +469,29 @@ paths:
...
+      parameters:
+      - $ref: '#/components/parameters/personaKeyPathParam'
+      tags:
+        - Admin APIs
+      security:
+        - bearerAdminAuth: []
+      responses:
+        204:
+          description: Empty response on success
+        401:
```
Comment (mikelax) [inline server/the-usher-openapi-spec.yaml:486]: @k1ch small nit, should the response codes be listed in ascending order? If you want to fix then also make the change for the spec added in #89 .

**Review by mikelax — DISMISSED** [review]
@k1ch nice job on this and with covering the cascade delete tests.

File: server/the-usher-openapi-spec.yaml (hunk 1 above)
Comment (k1ch) [reply server/the-usher-openapi-spec.yaml:486]: Yes, it should. I will update.

**Comment by mikelax** [comment]
@k1ch if you are going to make one more change for this PR per the comment above, at the same time do you want to resolve the [code scanning items](https://github.com/DMGT-TECH/the-usher-server/security/code-scanning) related to the useless imports?

**Review by mikelax — APPROVED** [review]

**Pull request merged**